core/trino-main/src/test/java/io/trino/TestSession.java

```diff
@@ -61,4 +69,53 @@ public void testAddSecondCatalogProperty()
                         "first_property", "some_value",
                         "second_property", "another_value")));
     }
+
+    @Test
+    public void testCreateViewSession()
+    {
+        final var catalog = Optional.of("test_catalog");
+        final var schema = Optional.of("test_schema");
```

Comment:
We avoid using `var`.
Please look at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#code-style to get an idea of the overall development guidelines